www/front_src/src/translations/index.js

```diff
@@ -5,17 +5,24 @@ const translationService = axios("internal.php?object=centreon_i18n&action=trans
 const userService = axios("internal.php?object=centreon_topcounter&action=user");
 
 export default function setTranslations(store, callback) {
-    const localePromise = userService.get();
-    const translationsPromise = translationService.get();
-    Promise
-        .all([localePromise, translationsPromise])
-        .then(response => {
-            let locale = response[0].data.locale;
-            locale = locale != null ? locale.slice(0,2) : navigator.language;
-            const translations = response[1].data
-            syncTranslationWithStore(store);
-            store.dispatch(loadTranslations(translations));
-            store.dispatch(setLocale(locale));
-            callback();
-        });
+  const localePromise = userService.get();
+  const translationsPromise = translationService.get();
+
+  Promise
+    .all([localePromise, translationsPromise])
+    .then(response => {
+      let locale = response[0].data.locale;
+      locale = locale !== null ? locale.slice(0,2) : navigator.language;
+      const translations = response[1].data;
+      syncTranslationWithStore(store);
+      store.dispatch(loadTranslations(translations));
+      store.dispatch(setLocale(locale));
+      callback();
+    })
+    .catch((error) => {
+      if (error.response.status === 401) {
+        // redirect to login page
+        window.location.href = 'index.php?disconnect=1';
```

Comment:
It's work for all parth this href ?

Reply:
yes, because a base href is set in index.html